src/main/java/org/jitsi/videobridge/rest/LongPollingServlet.java

```diff
@@ -38,7 +59,55 @@ protected void doGet(
         throws IOException,
                ServletException
     {
-        // TODO Auto-generated method stub
-        response.setStatus(HttpServletResponse.SC_NOT_FOUND);
+        System.out.println("BB: IN LONG POLLING");
+        Continuation continuation = ContinuationSupport.getContinuation(request);
```

Comment:
i think that i did look into the async context stuff, but ended up going down the continuation route--i don't remember what the reason was.  but i'll work on moving it over to the async servlet stuff.